spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/stream/SkipperStreamDeployer.java

```diff
@@ -380,8 +382,20 @@ public AppStatus getAppStatus(String id) {
 
 	@Override
 	public RuntimeEnvironmentInfo environmentInfo() {
-		// TODO To fix when https://github.com/spring-cloud/spring-cloud-skipper/issues/392
-		throw new UnsupportedOperationException("EnvironmentInfo is not supported for Skipper mode");
+		AboutInfo skipperInfo = skipperClient.info();
+		Resources<Deployer> deployers = skipperClient.listDeployers();
+		RuntimeEnvironmentInfo.Builder builder = new RuntimeEnvironmentInfo.Builder()
+				.implementationName(skipperInfo.getName())
+				.implementationVersion(skipperInfo.getVersion())
+				.platformApiVersion("")
+				.platformClientVersion("")
+				.platformHostVersion("")
+				.platformType("Skipper Managed")
+				.spiClass(SkipperClient.class);
+		for (Deployer d : deployers) {
+			builder.addPlatformSpecificInfo(d.getName(), d.getType());
```

Comment:
@tzolov I guess we will have an associated PR in Skipper to have the type `local` for the `default` platform.  Going ahead with the merge as this PR doesn't need to wait for it anyway.